pkg/sources/gitlab/gitlab.go

```diff
@@ -324,7 +324,7 @@ func (s *Source) scanRepos(ctx context.Context, chunksChan chan *sources.Chunk,
 				}
 				s.SetProgressComplete(i, len(repos), fmt.Sprintf("Repo: %s", repoURL), "")
 
-				path, repo, err := git.CloneRepoUsingToken(s.token, repoURL.String(), s.user)
+				path, repo, err := git.CloneRepoUsingToken(s.token, repoURL.String(), "placeholder")
```

Comment:
I think basic auth used to be supported on gitlab, so we should leave the option for a username. The on prem gitlab offering could still support it. Using "placeholder" if s.user is empty should work.

Reply:
makes sense. Updated to check if s.user is set in the event it isn't we can use "placeholder".